UnitTests/UnitTests.UWP/UnitTests.UWP.csproj

```diff
@@ -112,6 +112,9 @@
     <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform">
       <Version>6.2.12</Version>
     </PackageReference>
+    <PackageReference Include="Microsoft.Xaml.Behaviors.Uwp.Managed">
+      <Version>2.0.1</Version>
+    </PackageReference>
```

Comment:
Shouldn't this be included automatically with the inclusion of our Behaviors project?

Reply:
Indeed, this doesn't seem to be needed. I've removed it.